src/main/java/mpersand/Gmuwiki/global/config/security/SecurityConfig.java

```diff
@@ -36,7 +37,9 @@ protected SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .requestMatchers(CorsUtils::isPreFlightRequest).permitAll()
                 .antMatchers("/auth/**").permitAll()
                 .antMatchers("/email/**").permitAll()
-                .anyRequest().authenticated();
+                .antMatchers("/admin/notice/**").hasAuthority("ADMIN")
+                .antMatchers("/user/notice/**").hasAuthority("USER")
```

Comment:
권한을 url로 나눌거라면 
```suggestion
                .antMatchers("/admin/**").hasAuthority("ADMIN")
                .antMatchers("/user/**").hasAuthority("USER")
```
이런식으로 바꾸는게 좋을거 같아요

Reply:
넵 감사합니다